index/slashing/rpc/rpc.proto

```diff
@@ -4,7 +4,7 @@ package index.slashing.rpc;
 option go_package = "index/slashing/rpc";
 
 message Index {
-    string tipSetKey = 1;
+    string tip_set_key = 1;
```

Comment:
This might be an exception to the rule: should be `tipsetkey`.
Or at least ideally should be an exception, not sure if that's hard to make an exception in the linter.
If it's a headache, I would leave it as is... np.

Reply:
That was my interpretation, but if you want it all one word, that is fine with the linter.

Reply:
It will be `Tipsetkey` in Go. making the change now.